src/main/resources/view/MainWindow.fxml

```diff
@@ -44,7 +44,7 @@
                   <AnchorPane fx:id="taskListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
                </children>
             </VBox>
-            <AnchorPane fx:id="upcomingTaskPlaceholder" prefWidth="340" >
+            <AnchorPane fx:id="upcomingTaskPlaceholder" prefWidth="650" >
```

Comment:
You can't centralised by changing the width. It only works on your screen, it won't be centralised on other screens.